src/orders/src/main/java/mushop/orders/services/OrdersService.java

```diff
@@ -1,5 +1,7 @@
 package mushop.orders.services;
 
+import io.micrometer.core.instrument.DistributionSummary;
```

Comment:
I'm assuming these don't require any new additions in gradle?

Reply:
thats correct. Micrometer is part (implicit dependency) of [SpringBoot 2.3 ](https://spring.io/blog/2020/05/15/spring-boot-2-3-0-available-now)- we upgraded to  SpringBoot 2.3 a while back, which automatically gives us access to micrometer.